lib/utils.js

```diff
@@ -146,8 +146,10 @@ api.fetchInSecurityContext = async ({
       // measure until a zcap context is created
       isEqual(url['@context'], [
         constants.SECURITY_CONTEXT_V2_URL,
-        'https://w3id.org/security/suites/ed25519-2020/v1'
-      ])
+        'https://w3id.org/security/suites/ed25519-2020/v1',
+      ]) || isEqual(url['@context'], [
+      'https://w3id.org/webkms/v1',
```

Comment:
-1 to this -- we should only know about the zcap context here, not the webkms one. Ideally, this check should only be looking for context to be a string that matches sec/v2 or an array where the first element is the zcap context. Anything else should result in compaction being performed. I'm not sure if we need other temporary exceptions to that rule -- but if so, they should be marked as temporary until we get other things sorted out.

Exceptions should only be created if we know the compacted version can't work -- not to avoid compaction at the moment for optimization purposes.